srtcore/srt.h

```diff
@@ -148,6 +164,12 @@ typedef int32_t SRTSOCKET;
 // socket or a socket group.
 static const int32_t SRTGROUP_MASK = (1 << 30);
 
+#ifdef __cplusplus
+namespace srt {
+inline bool isgroup(SRTSOCKET sid) { return (int32_t(sid) & SRTGROUP_MASK) != 0; }
+}
+#endif
```

Comment:
C++ API (inline function) mixed with C API.

Reply:
I think it can be moved. I wanted to make it available for the users. We can just as well add this to the internals and later move it to the API.